Sources/Fuzzilli/Engines/FuzzEngine.swift

```diff
@@ -36,6 +36,9 @@ extension FuzzEngine {
                 if let aspects = fuzzer.evaluator.evaluate(execution) {
                     if fuzzer.config.enableInspection {
                         program.comments.add("Program may be interesting due to \(aspects)", at: .footer)
+                        if fuzzer.config.enableArgumentRandomization {
+                            program.comments.add("ARGS: \(fuzzer.runner.processArguments.joined(separator: " "))", at: .header)
+                        }
```

Comment:
Let me know if there is a better place for this block :). I was also consider putting this block in `Storage.swift` between the following lines instead:

https://github.com/googleprojectzero/fuzzilli/blob/48163c833b1ac42b05058e7a73256967a1f741c3/Sources/Fuzzilli/Modules/Storage.swift#L176-L177

Reply:
> Additionally I think we can just drop the change in Configuration.swift and always write the arguments in the .fuzzil.history, maybe "RUNNER ARGS: " to be a bit more verbose. WDYT?

Yep, that sounds good to me. I made the two changes you suggested :)